### PR TITLE
Allow devs to override the inputs

### PR DESCRIPTION
**Pull request by Zacthor**

### CODE REVIEW

**Review by ellisio — APPROVED** [review]
It looks good to me. Before making a PR to upstream, can you explain a little about why we want to be able to override this? That way we can poke holes in your theory, if there are any, before you send this upstream.

**Comment by Zacthor** [comment]
> It looks good to me. Before making a PR to upstream, can you explain a little about why we want to be able to override this? That way we can poke holes in your theory, if there are any, before you send this upstream.

If you take a look at `DateRangePickerInput` the html inside is very prescriptive and hard to manipulate. Our designs have very different inputs for mobile and desktop and what is currently available for input will not support them both.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-dates",
+  "name": "@fluidtruck/react-dates",
   "version": "21.8.0",
```
Comment (ellisio) [inline package.json]: Should we take this down to `0.0.1` since this is our own version?

File: src/components/DayPicker.jsx [hunk 2]
```diff
@@ -197,6 +197,8 @@ export const defaultProps = {
   dayAriaLabelFormat: undefined,
 };
 
+const SINGLE_LETTER_HEADER = 'sngl'
```
Comment (ellisio) [inline src/components/DayPicker.jsx]: Can we make this `single`?

**Review by ellisio — APPROVED** [review]
Just two questions, not blockers.

**Review by kmiller9393 — APPROVED** [review]

**Review by oakley808 — APPROVED** [review]

**Pull request merged**